cli/Cargo.toml

```diff
@@ -106,6 +106,12 @@ async-trait = { workspace = true }
 insta = { workspace = true }
 test-case = { workspace = true }
 testutils = { workspace = true }
+gix = { workspace = true, features = [
```

Comment:
nit: insert before `insta`. The list appears to be sorted (except for internal deps.)